README.md

```diff
@@ -1,80 +1,190 @@
 # Tweeteor
-### meteor-workshop
+Welcome to this Meteor workshop made for BEKK. This is the second part of the workshop, so if you haven't completed the [first part](https://github.com/EirikV/meteor-workshop/tree/chat-solution) yet, we strongly recommend that you do.
 
-This is a workshop for JS-Platform at Bekk Consulting.
+In this part you are jumping into some existing code for a social microblog application called Tweeteor. Tweeteor revolutionizes the way social microblogs work by fetching random images from Instagram that matches your hashtags. Wonder how it looks like? Go [check it out](http://therealtweeteor.meteor.com/). Feel free to create an account and write some tweets.
 
-The objective is to learn about Meteor (not to be confused with meteorites or comets) by building Tweeteor.
-Tweeteor is a fun new micro-blog that combines your hashtags with instagram pictures.
+## Some resources to help you along
+* **[The full Meteor documentation](http://docs.meteor.com/#/full/quickstart)**
+* **[Atmospherejs.com - meteor packages](https://atmospherejs.com/)**
+* **[The iron router guide](http://iron-meteor.github.io/iron-router/)**
+* **[Collection FS github page (used for image upload)](https://github.com/CollectionFS/Meteor-CollectionFS)**
 
-[Meteor documentation](http://docs.meteor.com/#/full/)
+### Tasks
```

Comment:
Burde kanskje være ```##Tasks``` siden den ikke er en undertittel av "Some resources to help you along"